airbyte-api/src/main/openapi/config.yaml

```diff
@@ -52,6 +52,8 @@ tags:
     description: Connection between sources and destinations.
   - name: health
     description: Healthchecks
+  - name: migration
```

Comment:
i would suggest we do not use the word "migrate" here. Migrate is describing what we will be using the endpoints _for_ not what the endpoints actually return  _is_. 

In the API I would suggest we use a namespace such as: `data`, `archive` or something like that.

Reply:
Was looking for what other companies do. Cloudera Manager call [this namespace](https://docs.cloudera.com/cloudera-manager/7.0.1/concepts/topics/cm-api-backup-restore-configuration.html) `deployment` which i like well enough.